src/PaginationLink.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  next: PropTypes.bool,
+  previous: PropTypes.bool,
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+};
+
+const defaultProps = {
+  tag: 'a',
+};
+
+const PaginationLink = (props) => {
+  const {
+    children,
+    className,
+    next,
+    previous,
+    tag: Tag,
+    ...attributes,
+  } = props;
+
+  const classes = classNames(
+    className,
+    'page-link'
+  );
+
+  return (
+    <Tag {...attributes} className={classes}>
```

Comment:
For prev/next, I don't think it will ever require children unless someone wants to override the carets. Could you setup rendering to be: if prev/next, include spans and render children instead of carets vs rendering children before the prev/next content.